frontend/common/constants.ts

```diff
@@ -212,6 +212,12 @@ export default {
     primary: '#6837fc',
     white: '#ffffff',
   },
+  contentTypes: {
+    environment: 30,
```

Comment:
Are these hardcoded ids ? Not clear what this is. If they are ids, does this differ per environment or are they somehow fixed.

Reply:
These are the IDs of the content types of those 4 entities, they do not change.
<img width="328" alt="Screen Shot 2023-12-14 at 16 32 38" src="https://github.com/Flagsmith/flagsmith/assets/41410593/d04125dc-6364-42e7-9960-35eb320173e1">